### PR TITLE
UpdateReadme.md

### PR DESCRIPTION
**Pull request by gujralsanyam22**
UpdateReadme.md

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -72,7 +72,8 @@ By default, GPU support is built if CUDA is found and ``torch.cuda.is_available(
 It's possible to force building GPU support by setting ``FORCE_CUDA=1`` environment variable,
 which is useful when building a docker image.
 
-Image Backend
+
+**Image Backend**
```
Comment (fmassa) [inline README.rst:76]: From rendering it on markdown, there doesn't seem to be any difference by making the titles bold, as it's already done by default. Can you revert those changes?

File: README.rst [hunk 2]
```diff
@@ -89,12 +90,12 @@ Torchvision currently supports the following image backends:
...
-.. _Pillow-SIMD : https://github.com/uploadcare/pillow-simd
-.. _accimage: https://github.com/pytorch/accimage
-.. _libjpeg: http://ijg.org/
-.. _libjpeg-turbo: https://libjpeg-turbo.org/
+1.) _libpng : http://www.libpng.org/pub/png/libpng.html
+2.) _Pillow : https://python-pillow.org/
+3.) _Pillow-SIMD : https://github.com/uploadcare/pillow-simd
+4.) _accimage: https://github.com/pytorch/accimage
+5.) _libjpeg: http://ijg.org/
+6.) _libjpeg-turbo: https://libjpeg-turbo.org/
```
Comment (fmassa) [inline README.rst:98]: This doesn't seem to render correctly, can you revert this change? 
image

File: README.rst [hunk 3]
```diff
@@ -123,17 +124,17 @@ so make sure that it is also available to cmake via the ``CMAKE_PREFIX_PATH``.
...
+**Contributing**
 ============
 We appreciate all contributions. If you are planning to contribute back bug-fixes, please do so without any further discussion. If you plan to contribute new features, utility functions or extensions, please first open an issue and discuss the feature with us.
 
-Disclaimer on Datasets
+**Disclaimer on Datasets**
 ======================
 
-This is a utility library that downloads and prepares public datasets. We do not host or distribute these datasets, vouch for their quality or fairness, or claim that you have license to use the dataset. It is your responsibility to determine whether you have permission to use the dataset under the dataset's license.
+This is a utility library that downloads and prepares the  public datasets. We do not host or redistribute these datasets, vouch for their quality or fairness, or claim that you have license to use the dataset. It is your responsibility to determine whether you have permission to use the dataset under the dataset's license.
```
Comment (fmassa) [inline README.rst:138]: I think the `the` is unnecessary.

The `redistribute` I'm not sure, a Google search s[eemed to look as if `distribute` could be ok](https://wikidiff.com/distribution/redistribution).

File: README.rst [hunk 4]
```diff
@@ -1,4 +1,4 @@
-torchvision
+**Torchvision**
```
Comment (fmassa) [inline README.rst:1]: We haven't been consistent with the capitalization of torchvision in the past. We've written it both as torchvision as well as TorchVision. From this perspective, I would maybe leave this as is, but I'm bringing this to discussion with @dongreenberg

**Review by fmassa — CHANGES_REQUESTED** [review]
Thanks for the PR!

I've left a few comments, and I believe there is one issue with formatting. Would you be willing to try to fix it?

**Comment by t-vi** [comment]
Context: https://blog.domenic.me/hacktoberfest/
Same similar thing in https://github.com/pytorch/text/pull/1018

**Pull request closed without merging**